compiler/ballerina-lang/src/main/java/org/ballerinalang/project/Project.java

```diff
@@ -35,15 +35,15 @@
      * Returns true if the module exists in the project.
      *
      * @param moduleId Module Id
-     * @return
+     * @return boolean
```

Comment:
In java doc comments the @returns specify what it returns not the type of return value, the type of the return value is alredy in the function signature.

In this instance it's something like `module exists in the project`

Reply:
Updated the Javadocs.